lib/service/database.js

```diff
@@ -249,6 +249,10 @@ function checkCacheTableQuery(targetTableName) {
     return 'SELECT CDB_CheckAnalysisQuota(\'' + targetTableName + '\')';
 }
 
+function analyzeTableQuery(targetTableName) {
+    return 'ANALYZE ' + targetTableName;
```

Comment:
What is it intended to accomplish?? I mean, we aren't saving `ANALYSE` output anywhere.